dbt_platform_helper/addon-plans.yml

```diff
@@ -145,6 +145,14 @@ rds-postgres:
     multi_az: true
     instance: db.t3.small
 
+  # 2v CPU, 2GB RAM, 100GB Storage, multi AZ
+  small-high-io:
+    volume_size: 100
+    multi_az: true
+    instance: db.t3.small
+    storage_type: io1
+    iops: 40000
+
```

Comment:
Why have these been added? We're using the plans.yml from the extensions module for terraform aren't we?

Reply:
(I guess I'm getting at we should have this plans.yml file in one place. Not sure if this is the correct place or the terraform-platform-modules repo. Or elsewhere.)

Reply:
Yes, I raised an issue about this.  They're needed for validation and for the old copilot pathway.  We can remove from here when we retire the legacy solution.

Reply:
OK. Let's do that as soon as possible.